src/fsm_paper_speed_comparison.cpp

```diff
@@ -36,14 +36,21 @@ int main(int argc, char **argv){
 
   //Label the ocean cells. This is a precondition for using
   //`GetDepressionHierarchy()`.
+
+  int ocean_level_exists = 0;
   #pragma omp parallel for
   for(unsigned int i=0;i<label.size();i++){
     if(topo.isNoData(i) || topo(i)==ocean_level){ //Ocean Level is assumed to be lower than any other cells (even Death Valley)
       label(i) = dh::OCEAN;
       wtd  (i) = 0;
     }
+    if(topo(i) == ocean_level)
+      ocean_level_exists = 1;
   }
 
+  if(ocean_level_exists == 0)
+    throw std::runtime_error("There are no ocean_level cells in this topography!");
+
```

Comment:
Throwing an error if no cells with ocean_level exist. 
I've made the mistake of using an invalid ocean_level more than once, and the code appears to run, but outputs topography with 0 water on it. It's not immediately obvious why this happens, so the error should be an improvement (not sure if there's a better way to check the array for any ocean_level cells).

Reply:
It'd be preferable to do this as a separate PR, since it's easy to verify and merge. I'd suggest setting `ocean_level_exists` to be of type `bool` and initially equal to `false`. You can then use `ocean_level_exists = true` later and simplify the `if` clause on Line 51.

Reply:
Sorry for the silly question, but do I need to create a new branch in order to have a separate pull request for this?